articles/azure-stack/azure-stack-update-1903.md

```diff
@@ -107,24 +107,31 @@ The following are post-installation known issues for this build version.
+- Azure Stack Dashboard Feedback button is not responding/redirecting. The root cause in the fact that a recent change in Azure was not taken into account in Azure Stack. This change consists of the change of the name of the setting used by the Feedback button from **giveFeedback** to **submitAnIdea**. Release 1904 will have this issue corrected.
+
+<!-- 2930820 - IS ASDK -->
+- In both the administrator and user portals, if you search for "Docker," the item is incorrectly returned. It is not available in Azure Stack. If you try to create it, a blade with an error indication is displayed.
 
-<!-- 2931230 – IS  ASDK --> 
-- Plans that are added to a user subscription as an add-on plan cannot be deleted, even when you remove the plan from the user subscription. The plan will remain until the subscriptions that reference the add-on plan are also deleted. 
+<!-- 2931230 – IS  ASDK -->
+- Plans that are added to a user subscription as an add-on plan cannot be deleted, even when you remove the plan from the user subscription. The plan will remain until the subscriptions that reference the add-on plan are also deleted.
 
-<!-- TBD - IS ASDK --> 
+<!-- TBD - IS ASDK -->
 - The two administrative subscription types that were introduced with version 1804 should not be used. The subscription types are **Metering subscription**, and **Consumption subscription**. These subscription types are visible in new Azure Stack environments beginning with version 1804 but are not yet ready for use. You should continue to use the **Default Provider** subscription type.
 
-<!-- 3557860 - IS ASDK --> 
+<!-- 3557860 - IS ASDK -->
 - Deleting user subscriptions results in orphaned resources. As a workaround, first delete user resources or the entire resource group, and then delete the user subscriptions.
 
-<!-- 1663805 - IS ASDK --> 
-- You cannot view permissions to your subscription using the Azure Stack portals. As a workaround, use [PowerShell to verify permissions](/powershell/module/azs.subscriptions.admin/get-azssubscriptionplan).
+<!-- 1663805 - IS ASDK -->
+- You cannot view permissions to your subscription using the Azure Stack portals. As a workaround, use [PowerShell to verify permissions](https://docs.microsoft.com/en-us/powershell/module/azurerm.resources/get-azurermroleassignment).
 
 <!-- Daniel 3/28 -->
-- In the user portal, when you navigate to a blob within a storage account and try to open **Access Policy** from the navigation tree, the subsequent window fails to load.
+- In the user portal, when you navigate to a blob within a storage account and try to open **Access Policy** from the navigation tree, the subsequent window fails to load. To work around this issue, you can use the following PowerShell cmdlets for creating, retrieving, setting, and deleting access policies, respectively:
```

Comment:
Already reported by another customer and the fix has been submitted. Thank you again!